frontend/src/components/CANs/CANTypes.d.ts

```diff
@@ -17,13 +15,15 @@ export type CAN = {
     id: number;
     nick_name?: string;
     number: string;
-    obligate_by?: Date;
+    obligate_by?: number;
```

Comment:
@rajohnson90 didn't think this was a Date

Reply:
It isn't a date. Already covered this on collab but just writing it down so it's documented.